Library/Homebrew/livecheck/strategy/gnome.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Homebrew
+  module Livecheck
+    module Strategy
+      module Gnome
+        module_function
+
+        NICE_NAME = "GNOME"
+
+        # Formulae that do not use GNOME's "even-numbered minor is stable" scheme
```

Comment:
This may have some overlap with `brew audit` which has a similar list. Would be good to combine them.

Reply:
👍🏻 but even then I think it would be good to integrate with the `audit` list that's checking the same thing.

Reply:
I just checked the `GNOME_DEVEL_ALLOWLIST` in `dev-cmd/audit.rb` and it references the same formulae, so I imagine we're good here.

The main differentiator is that the `Gnome` strategy's `DEV_VERSION_ALLOWLIST` used the package name (e.g., `libart_lgpl`) rather than the formula name (e.g., `libart`). `libart` is the exception and the others have an identical formula name and package name.